assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@common/Button';
+import Table from '@common/Table';
+import PageHeader from '@common/PageHeader';
+import Modal from '@common/Modal';
+import Tooltip from '@common/Tooltip';
+import Banner from '@common/Banners/Banner';
+import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
+
+const USER_CREATE_ROUTE = '/users/new';
```

Comment:
I would put this simply in the `navigate`, we use it once, and it is even easier to see what happens.
What do you think?